tests/kernel/wave/wave_e2e_test.py

```diff
@@ -50,9 +49,9 @@ def test(
         {
             M: shape[0],
             N: shape[1],
-            BLOCK_M: 16,
-            BLOCK_N: 16,
-            ELEMS_PER_THREAD: 4,
+            BLOCK_M: 1,
+            BLOCK_N: 128,
+            ELEMS_PER_THREAD: 2,

```

Comment:
As discussed offline, it's equivalent to `min(N / 64, 8)`, right? Will it work if we write it this way?

Or, alternatively, we can try to define `ELEMS_PER_THREAD = `min(N / 64, 8)`` at the beginning.

Reply:
This will work, we do not have to load 8 elements. We can choose how many elements we want to load. But right now since the N dim is small (128) and we have 64 threads at least(size of warp), we can at most use 2 element per thread.